### ISSUE TITLE
Unable to sign up via email to restore purchases 

### ISSUE THREAD

**Issue by vsobris** [hymns-for-worship/issues#69]
When I attempt to create an account by signing up with email so I can restore my list data and purchases, nothing happens when I try to submit the information from the sign up form. The submit button is labeled “log in with email”, but nothing happens. I have tried several times and confirmed the passwords match.

**Comment by rodydavis** [comment]
Fixed on 2.3.0!

**Comment by vsobris** [comment]
The wording on the submit button has changed and it seems to actually try to process the information now, but it just sits with the loading circle on the submit button when trying to submit registration information.

**Comment by rodydavis** [comment]
iOS 2.3.1 is not rolled out, apple is still reviewing it. But for android it is out.

The issue was it was not showing the error message and just loading. So you should see an update soon.

**Comment by vsobris** [comment]
My iOS version is 16.6 and my HFW app version is 2.3.0+716.

**Comment by rodydavis** [comment]
Yep, update should be out for everyone on iOS in a day or so after apple approves

**Comment by vsobris** [comment]
I have 2.3.1 now and I still can’t create an account. When I tap “create account” it shows a screen of the back end for a few seconds and then goes back to the registration form.

**Comment by vsobris** [comment]
See attached screenshot for the issue mentioned in my last email.

**Comment by vsobris** [comment]
I am still having this issue. When attempting to create an account, I receive a screen similar to what I showed you in my previous email. Error code 400, status code 400. I then tried to do a password recovery. I received the same codes after submitting the form for password recovery in the app, but afterwards I received an email with a password recovery link. The form via the link was submitted successfull, but I still can’t log in. I receive a message saying “failed to authenticate” at the login screen.
